src/components/routes/Trips/StartedTripCard.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+
+import StartedTripCardContainer from './StartedTripCard.container';
+import LazyImage from 'components/shared/LazyImage';
+import { formatSingleDate } from 'utils/formatDate';
+import Divider from 'components/shared/Divider';
+import ListItem from 'components/shared/ListItem';
+import BeeLink from 'components/shared/BeeLink';
+import { AppConsumer, AppConsumerProps, ScreenType } from 'components/App.context';
+import Svg from 'components/shared/Svg';
+import Button from 'components/shared/Button';
+import { Booking } from 'networking/bookings';
+import { stringifyQueryString } from 'utils/queryParams';
+import { Listing } from 'networking/listings';
+
+interface Props {
+  trip: Booking;
+}
+
+const StartedTripCard = ({ trip }: Props) => {
+  const { checkInDate, checkOutDate, listing, numberOfGuests } = trip;
+  const { idSlug } = listing;
+  const streetAddress = (listing.addressLine1 || '').concat(listing.addressLine2 ? `, ${listing.addressLine2}` : '');
```

Comment:
Is StartedTripCard for bookings where `status` is `started`? If so, I don't you'll have the address fields unless you're an admin user